libraries/AP_Generator/AP_Generator_IE_FuelCell.h

```diff
@@ -0,0 +1,178 @@
+#pragma once
+
+#include "AP_Generator_Backend.h"
+
+#ifndef GENERATOR_ENABLED
+#define GENERATOR_ENABLED !HAL_MINIMIZE_FEATURES
+#endif
+
+#if GENERATOR_ENABLED
+
+#include <AP_Param/AP_Param.h>
+#include <AP_SerialManager/AP_SerialManager.h>
+#include <GCS_MAVLink/GCS.h>
+
+#define TERM_BUFFER 12          // Max length of term we expect
```

Comment:
This doesn't belong in the library's interface.  Just inline it.

Similarly elsewhere